ui/components/multichain/pages/all-connections/connection-list-item.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  AlignItems,
+  BackgroundColor,
+  BlockSize,
+  Display,
+  FlexDirection,
+  FlexWrap,
+  IconColor,
+  JustifyContent,
+  TextAlign,
+  TextColor,
+  TextVariant,
+} from '../../../../helpers/constants/design-system';
+import { useI18nContext } from '../../../../hooks/useI18nContext';
+import {
+  AvatarFavicon,
+  BadgeWrapper,
+  Box,
+  Icon,
+  IconName,
+  IconSize,
+  Text,
+} from '../../../component-library';
+import { getURLHost } from '../../../../helpers/utils/util';
+
+export const ConnectionListItem = ({ key, connection }) => {
```

Comment:
We'll need an `onClick` prop.

Reply:
added in [24b0b50](https://github.com/MetaMask/metamask-extension/pull/21332/commits/24b0b50c8a97033230be9d2692567a166df8aded)